ref_ts/static/ref_ts/datepicker/css/datepicker.css

```diff
@@ -1,182 +0,0 @@
-/*!
```

Comment:
@zhiyuli Likewise for datepicker and other JS libraries.  We need to think carefully about how to integrate these.  I would much rather not allow each resource type to import arbitrary JS libraries and accompanying CSS will-nilly.